README.md

```diff
@@ -67,7 +67,7 @@ Or you may also ask questions on StackOverflow at:
 
   The latest generated libraries for any Google API is available automatically
   from
-  https://developers.google.com/resources/api-libraries/download/<API>/<VERSION>/cpp
+  https://developers.google.com/resources/api-libraries/download/\<API\>/\<VERSION\>/cpp
```

Comment:
This doesn't get rendered quite right:
![image](https://user-images.githubusercontent.com/11778530/54446613-95317b00-4704-11e9-9159-71a6d3196e94.png)

Reply:
Fixed, thanks!